docs/client.rst

```diff
@@ -894,6 +894,27 @@ The request will be sent in the next ``run_session`` function call.
 
 ------
 
+.. code-block:: c
+    bool uxr_prepare_output_stream_fragmented(uxrSession* session, uxrStreamId stream_id, uxrObjectId datawriter_id,
+                                  struct ucdrBuffer* ub, size_t topic_size, uxrOnBuffersFull flush_callback);
+
+Requests a writing into a specific output stream.
+This function will initialize an ``ucdrBuffer`` struct where a topic of ``topic_size`` size must be serialized.
+Whether the necessary gap for writing a ``topic_size`` bytes into the reliable stream, the returned value is ``true``, otherwise ``false``.
```

Comment:
```suggestion
If there is sufficient space for writing ``topic_size`` bytes into the reliable stream, the returned value is ``true``, otherwise it is ``false``.
```